fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/SheetHorizontalItemAdapter.kt

```diff
@@ -46,7 +46,7 @@ class SheetHorizontalItemAdapter(private val context: Context, items: ArrayList<
 
         fun setBottomSheetItem(item: SheetItem) {
             if (item.bitmap != null) {
-                listItemView.update(item.title, context.createImageView(item.bitmap))
+                listItemView.update(item.title, context.createImageView(item.bitmap!!))
```

Comment:
How was this working earlier?

Reply:
Have made sheet items properties mutable now so that consumer can change and update the item